bin/periphery/src/stats.rs

```diff
@@ -86,20 +112,54 @@ impl StatsClient {
     self.system.refresh_memory();
     self.system.refresh_processes(ProcessesToUpdate::All, true);
     self.disks.refresh();
+    self.network.refresh();
   }
 
   fn refresh_lists(&mut self) {
     self.disks.refresh_list();
+    self.network.refresh_list();
   }
 
   pub fn get_system_stats(&self) -> SystemStats {
     let total_mem = self.system.total_memory();
     let available_mem = self.system.available_memory();
+
+    let mut total_ingress: f64 = 0.0;
```

Comment:
The totals should use the numerical type returned by eg `network.received()`. The cast to f64 should happen only at the end.

Reply:
I changed to u64 and converted later to f64, but I don't know if this is what you had in mind. Please let me know

Reply:
Perfect